apps/settings/js/modules/addon_manager.js

```diff
@@ -568,7 +568,7 @@ define(function(require) {
      * @access public
      * @memberOf AddonManager.prototype
      * @param {App} addon
-     * @returns {Promise String}
+     * @returns {(Promise|String)}
```

Comment:
`|` means the type can be either `Promise` or `String`. However the syntax we need here is "a promise resolved with a string". The closest syntax that I can find is `Promise.<String>`. What do you think?